source/_integrations/climacell.markdown

```diff
@@ -17,13 +17,11 @@ ha_platforms:
   - weather
 ---
 
-The ClimaCell integration allows you to obtain weather, air quality, pollen, and fire information from the [ClimaCell API](https://www.climacell.co/weather-api/).
-
-## Obtain an API key
-
-You can obtain a free API key by signing up with [ClimaCell](https://developer.climacell.co/sign-up?_ga=2.137889264.1908484805.1591592950-510691096.1591288729).
+<div class='note warning'>
```

Comment:
I think we should just add a redirect instead (`source/_redirects`).

If you do so, please be sure to rebase this PR before doing that, as the redirect file has massively changed this week.